pkg/handlers/s3_handler.go

```diff
@@ -214,4 +226,28 @@ func (h *s3Handler) OnDelete(ns string, vc kvcv1.VolumeConfig, controllerRef met
 			podClient.Delete(ns, resource.GetName())
 		}
 	}
+
+	// Delete the annotation for the node
```

Comment:
s/annotation/label. There is `annotation` field in k8s. This comment will be confusing.